rust_snuba/src/processors/generic_metrics.rs

```diff
@@ -240,6 +246,17 @@ where
     T: Parse + Serialize,
 {
     let payload_bytes = payload.payload().context("Expected payload")?;
+    let use_case: MessageUseCase = serde_json::from_slice(payload_bytes)?;
+    let killswitch_config = get_str_config("key");
```

Comment:
Is `key` the name of the runtime config? If so, we should name it to something that is more descriptive

Reply:
That was a placeholder name, sorry! I think we would need to create the actual runtime config first